src/_components/Assets/AssetsContainer.jsx

```diff
@@ -53,10 +54,21 @@ export const mapStateToProps = ({ assets, assetTypesList, assetModelsList, selec
     isLoading,
     filterData: createFilterData(assetTypes, assetModels),
     activePage,
-    selected
+    selected,
+    assetStatus
   };
 };
 
+export const AssetStatus = connect(mapStateToProps, {
```

Comment:
The reason I did this was to cater for a scenario where a user would come from let's say the `/assets/damaged` or `/assets/available` url to the `/assets` url. Since we are using the same component, it will not trigger a remount, thus the logic for checking whether the status of assets fetched would not be triggered and the appropriate assets would not be loaded.